example/openapi/api-docs.yaml

```diff
@@ -1035,331 +1361,458 @@ components:
-        _commitId:
+          readOnly: true
+        _modified:
           type: string
-      additionalProperties:
-        type: object
-    CommitsResponse:
+          readOnly: true
+      additionalProperties: true
+    RefsResponse:
+      required:
+      - refs
       type: object
       properties:
         messages:
           type: array
+          nullable: true
           items:
             type: string
+            nullable: true
         rejected:
           type: array
+          nullable: true
           items:
             $ref: '#/components/schemas/Rejection'
-        commits:
+        refs:
           type: array
           items:
-            $ref: '#/components/schemas/CommitJson'
-    CommitsRequest:
+            $ref: '#/components/schemas/RefJson'
+    RefsRequest:
+      required:
+      - refs
       type: object
       properties:
         source:
           type: string
+          nullable: true
         comment:
           type: string
-        commits:
+          nullable: true
+        refs:
           type: array
           items:
-            $ref: '#/components/schemas/CommitJson'
-    ProjectJson:
+            $ref: '#/components/schemas/RefJson'
+    CommitAddedJson:
       type: object
       properties:
-        projectType:
+        id:
           type: string
-        orgId:
+        type:
+          type: string
+        _docId:
+          type: string
+          readOnly: true
+      readOnly: true
+    CommitDeletedJson:
+      type: object
+      properties:
+        previousDocId:
+          type: string
+          readOnly: true
+        id:
           type: string
         type:
           type: string
-        name:
+      readOnly: true
+    CommitJson:
+      required:
+      - id
+      type: object
+      properties:
+        source:
+          type: string
+          readOnly: true
+        comment:
           type: string
+          readOnly: true
+        deleted:
+          type: array
+          readOnly: true
+          items:
+            $ref: '#/components/schemas/CommitDeletedJson'
+        added:
+          type: array
+          readOnly: true
+          items:
+            $ref: '#/components/schemas/CommitAddedJson'
+        updated:
+          type: array
+          readOnly: true
+          items:
+            $ref: '#/components/schemas/CommitUpdatedJson'
         id:
           type: string
         _projectId:
           type: string
+          readOnly: true
+        _refId:
+          type: string
+          readOnly: true
         _creator:
           type: string
+          readOnly: true
         _docId:
           type: string
-        _modifier:
+          readOnly: true
+        _created:
           type: string
-        _modified:
+          readOnly: true
+        _commitId:
           type: string
-        _created:
+          readOnly: true
+      additionalProperties: true
+    CommitUpdatedJson:
+      type: object
+      properties:
+        previousDocId:
```

Comment:
`previousDocId` doesn't start with an underscore but `_docId` does. Also applies to `CommitDeletedJson#previousDocId`.

Reply:
_ added